assets/svg/device-size-desktop-icon.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg name="google-sitekit-device-size-desktop" width="23" height="17" viewBox="0 0 23 17" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Does the `xml` tag at the top make a difference at all? If not maybe we should just remove it since it seems others don't have it.

I see the `name` was in the previous component as well but looks like it isn't used anywhere; perhaps we should remove this as well?

These comments apply to `assets/svg/device-size-mobile-icon.svg` too.

Reply:
@aaemnnosttv The XML declaration should not make a difference (unless we're using non-standard encoding) in-browser as the parser will strip this out before rendering. However, this may make a difference if you try to use the SVG in a non-browser environment (opening the SVG in a program to edit) which is why I try to XML declarations in unless we have a copy of the original file (Sketch/Illustrator file). I'd rather keep SVGs closer to source and process them on build (can minify SVGs via Webpack/SVGO to remove any extra attributes or declarations). As far as consistency goes (without the 2 newly added SVGs) 20 have XML declarations and 13 do not, adding the XML declaration follow consistency a bit more. Let me know how you feel about this, if we're going with no XML declarations I think we should remove them from the 20 SVGs that currently have them to avoid future confusion.

Reply:
> I'd rather keep SVGs closer to source and process them on build

That sounds like a good approach to me.

> As far as consistency goes (without the 2 newly added SVGs) 20 have XML declarations and 13 do not, adding the XML declaration follow consistency a bit more. Let me know how you feel about this, if we're going with no XML declarations I think we should remove them from the 20 SVGs that currently have them to avoid future confusion.

While I agree it would be good to have consistency here, it's not really in the scope of this issue to change so I'd prefer to revisit that in a separate issue (feel free to create that one if you'd like?).

Since the existing components didn't have the xml tag, let's not include them in the svg files for now since it seems it isn't required.